### ISSUE TITLE
RocketTheme\Toolbox\Event\Event  shows deprecation warning. What replacement to use?

### ISSUE THREAD

**Issue by pamtbaau** [getgrav/grav#3113]
When using `RocketTheme\Toolbox\Event\Event`, the IDE shows a warning about its deprecation.
> Event classes will be removed in the future. Use PSR-14 implementation instead.

```
public function onPageInitialized(Event $event) { ... }
```
Which event class should instead be imported?.

**Comment by mahagr** [comment]
Yes, we are aware of this. However we need to create new events that follow PSR-14 first, so it's more 2.0 thing...
